### ISSUE TITLE
rems showing up in the "Edit Later" even though they do not have the power up tag

### ISSUE THREAD

**Issue by brushplain** [remnoteio/remnote-issues#1081]
The "Edit Later" power-up/rem-page, accessed via sidebar persistently shows that I have 3 rems tagged with the "Edit Later" power-up, even though this is not the case. Since there are no rems actually tagged with the "Edit Later" power-up, I cannot remove the tag. This results in the annoyance of the "Edit Later" always telling me that I have 3 rems to edit later. 

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
I use Mac OS. I think I'm on remnote version 1.3
<img width="1440" alt="Screen Shot 2021-04-02 at 12 28 41 PM" src="https://user-images.githubusercontent.com/61999258/113390297-107b2d80-93af-11eb-8bbd-db4885e87fcd.png">

**Smartphone (please complete the following information):**

**Additional context**

*Note*: Please contact us about account-specific issues directly at support@remnote.io.

**Comment by drkvj** [comment]
Are you able to delete the tagged rem search portal? Maybe that can reset this issue

**Comment by brushplain** [comment]
No, I'm not able to delete the tagged rem search portal. When I try to delete the portal it instantly reappears. Also, somewhat unrelated, I switched to the paid "pro" account because I want to see remnote improve over time!

**Comment by drkvj** [comment]
Note: deleting the knowledge base and importing a backup brought back the done editing button and fixed this.

**Comment by drkvj** [comment]
related to #1052
